src/types.ts

```diff
@@ -676,6 +706,11 @@ export interface Payment extends Omit<DbPayment, "id"> {
   id: number;
 }
 
+export enum PermissionMethodLiquid {
+  LIQUID_SIGNSCHNORR = "liquid/signSchnorr",
+  LIQUID_GETPUBLICKEY = "liquid/getPublicKey",
```

Comment:
maybe we should rename to `LIQUID_GETADDRESS` ? (getPublicKey -> getAddress)

Reply:
fixed in https://github.com/getAlby/lightning-browser-extension/pull/2583